src/backend/metal/MetalBackend.h

```diff
@@ -168,6 +168,11 @@ namespace metal {
         public:
             RenderPass(RenderPassBuilder* builder);
             ~RenderPass();
+
+            uint32_t GetAttachmentFirstSubpass(uint32_t attachmentSlot);
```

Comment:
With the comments about storing this in RenderPassBase, changes to the file would disappear.